components/sup/src/manager/mod.rs

```diff
@@ -690,6 +691,22 @@ impl Manager {
             None => None,
         };
         loop {
+            if feat::is_enabled(feat::TestExit) {
+                if let Ok(exit_file_path) = env::var("HAB_FEAT_TEST_EXIT") {
+                    if let Ok(mut exit_code_file) = File::open(&exit_file_path) {
+                        let mut buffer = String::new();
+                        exit_code_file
+                            .read_to_string(&mut buffer)
+                            .expect("couldn't read");
+                        if let Ok(exit_code) = buffer.lines().next().unwrap_or("").parse::<i32>() {
+                            fs::remove_file(&exit_file_path).expect("foo");
```

Comment:
We could be a little more detailed than simply "foo" 😄

Reply:
Ha ha, I thought I updated that. Good catch.